examples/wearable/UIComponents/contents/navigationelements/index.html

```diff
@@ -31,18 +31,13 @@ <h2 class="ui-title">
 					</a>
 				</li>
 				<li>
-					<a href="indexscrollbar/index.html">
-						IndexScrollBar
+					<a href="scrollbar/index.html">
+						Scroll bar
 					</a>
 				</li>
 				<li>
-					<a href="fastscroll.html">
-						Fast Scroll
-					</a>
-				</li>
-				<li>
-					<a href="sectionchanger/index.html">
```

Comment:
Shouldn't we remove this file? Is it used somewhere else?

Reply:
no it is not used, I don't see section changer in guides too. I'll remove it

Reply:
Yes, I think it's better to move it etc since not used files are usually candidates to remove.

Reply:
ok, done